### DISCUSSION TITLE
Diskussion: Möglichkeit zur Identifizierung von räumlichen Einheiten

### DISCUSSION THREAD

**Discussion by jhohlfeld** [open-dms/Open-DMS#3] [General]
Hallo Team,

wir bauen `osmtools`, um Daten von OpenStreetMap auszulesen.
Die Daten wollen wir benutzen, um räumliche Strukturen mit deren Geodaten abzubilden, um den Travelguide bzw. Datenmonitor für den Knowledge-Graph der DTAG räumlich einschränken zu können.

Die Frage ist immer noch,

1. wie wir aus den Daten nur räumliche Einheiten aus Deutschland herausfiltern (Polen, Niederlande etc. sind immer auch mit dabei wegen der angrenzenden Dependencies von Relations)
2. wie wir bestimmen, welche räumlichen Kreise überhaupt sinnvoll sind
3. wie wir die Datenqualität am Ende sicherstellen

Hierzu habe ich die Daten für Mecklenburg-Vorpommern analysiert: [stats.txt](https://github.com/open-dms/osmtools/files/13269020/stats.txt).

Es zeichnet sich ab, dass wir Gemeinden und Kreise anhand der Amtlichen Gemeindeschlüssel und Regionalkreisschlüssel identifizieren können:

<img width="312" alt="Bildschirmfoto 2023-11-06 um 12 38 03" src="https://github.com/open-dms/osmtools/assets/497141/01582a3b-3620-47e2-8ec9-ab67a59cab5c">

https://de.wikipedia.org/wiki/Amtlicher_Gemeindeschl%C3%BCssel

**Comment by jhohlfeld** [comment]
Umso besser: der Regionalschlüssel enthält bereits den AGS (Amtlicher Gemeindeschlüssel):
https://de.wikipedia.org/wiki/Amtlicher_Gemeindeschl%C3%BCssel#Regionalschl.C3.BCssel

**Comment by herrhelms** [comment]
bin stark dafür erstmal ausschliesslich die erwähnten gemeindeamtschlüssel daten und bounds/paths zu ziehen. Also auf gängige "adminstrative areas" beschränken. (e.g. `Stadt Braunschweig`, `Landkreis Braunschweig`) Zum späteren Zeitpunkt auf anderen Layern können wir immernoch nachträglich andere Regionen hinzufügen/von OSM erben. (e.g. `Braunschweiger Land`)

**Comment by bbannier** [comment]
https://www.xrepository.de/details/urn:de:bund:destatis:bevoelkerungsstatistik:schluessel:rs

**Comment by jhohlfeld** [comment]
Hier noch als Ergänzung der Wiki-Eintrag zu Boundary-Relations: https://wiki.openstreetmap.org/wiki/Relation:boundary
